index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Learn GitHub</title>
```

Comment:
first time doing this, pretty cool stuff
```suggestion
  <title>Learn GitHub</title>
  not really sure what I'm doing just yet, please be patient!
```